src/samplers/independent.cpp

```diff
@@ -28,63 +28,43 @@ by Melissa O’Neill.
 
 This is the most basic sample generator; because no precautions are taken to avoid
 sample clumping, images produced using this plugin will usually take longer to converge.
-In theory, this sampler is initialized using a deterministic procedure, which means
-that subsequent runs of Mitsuba should create the same image. In practice, when
-rendering with multiple threads and/or machines, this is not true anymore, since the
-ordering of samples is influenced by the operating system scheduler.
+this sampler is initialized using a deterministic procedure, which means that subsequent runs
+of Mitsuba should create the same image.
```

Comment:
Not sure that the removal of the previous text is 100% correct. When rendering things in parallel on the CPU, the order in which blocks are merged into the film *will* cause very small differences. You could add a note that these should be absolutely negligible, with relative errors on the order of the machine epsilon (:math:`6\cdot 10^{-8}`) in single precision.

Reply:
I see, we are talking about numerical precision uncertainties in this case right?

Reply:
I wouldn't call it uncertainty, just minor variation due to OS scheduling in the presence of arithmetic that is non-commutative due to builtin rounding.